src/plugins/teleport-plugin-react-base-component/utils.ts

```diff
@@ -96,14 +96,21 @@ export const generateConditionalNode = (
 ) => {
   const { reference, value } = node.content
   const conditionIdentifier = createConditionIdentifier(reference, accumulators)
+  const stateKey = reference.content.id
+  const stateIdentifier = accumulators.stateIdentifiers[stateKey]
```

Comment:
Hey @alexnm i am thinking if i should stop implementing this additional check in generators and close the PR, because when i am developing i felt if we are doing a check for state, we should do for prop definitions as well and then the check are increasing, As you mentioned here --> https://github.com/teleporthq/teleport-code-generators/issues/145#issuecomment-479809748 now even i fell i might be adding additional checks into the generators if the #41 is implemented. What you say ?

Reply:
@JayaKrishnaNamburu ok, let's close this as we have plenty of changes in release/0.6 already